pkg/utils/root_unix.go

```diff
@@ -5,7 +5,7 @@
 //go:build !windows
 // +build !windows
 
-package install
+package utils
```

Comment:
I'm not really in favour of a utils package, as it does not say much. I rather have a few small, more meaningful packages. Perhaps it could be called `permissions` or `perms`

Reply:
We can look at that in a later time.